src/docs/sdk/performance/index.mdx

```diff
@@ -18,7 +18,11 @@ Reference implementations:
 
 This section describes the options SDKs should expose to configure tracing and performance monitoring.
 
-Tracing is enabled by setting either one of two new SDK config options, `tracesSampleRate` and `tracesSampler`. If not set, both default to `undefined`, making tracing opt-in.
+Tracing is enabled by setting any of three SDK config options, `enableTracing`, `tracesSampleRate` and `tracesSampler`. If not set, these options default to `undefined`, making tracing opt-in.
```

Comment:
IMO this line is going to add to the confusion. I get what we're trying to say, you can now just set `enableTracing=True` but it reads more confusing that that. 

What if we re-worded it to something like

```suggestion
Tracing is enabled by setting `enableTracing` to True. Once Tracing is enabled you can fine-tune which transactions are set by either`tracesSampleRate` or `tracesSampler`. If not set, these options default to `undefined`, making tracing opt-in.
```

Reply:
We need to make it BC.

Reply:
> We need to make it BC.
BC?

@silent1mezzo with this wording it would be interpreted as a breaking change (ah just understood BC XD). we do not want users to be required to set `enableTracing`, Setting any of the 2 existing options implies `true` and ensures all existing config should continue to work.

Reply:
The existing copy makes clear that either way is fine, the proposed one suggests that you need to first set `enableTracing`. I'd stick with the existing copy.